docs/Radarr/Radarr-recommended-naming-scheme.md

```diff
@@ -57,15 +57,13 @@ This naming scheme is made to be compatible with the [New Plex Agent](https://fo
 
 ### Emby
 
-!!! warning "Emby is the only one who uses `=` for their ID, which isn't supported by the Starr apps at the moment.<br>Keep a eye on the following [PR#1386](https://github.com/TRaSH-Guides/Guides/pull/1386){:target="_blank" rel="noopener noreferrer"} when it does support it"
-
 ```bash
 {{ radarr['naming']['radarr-naming']['file']['emby'] }}
 ```
 
 ??? abstract "RESULTS: - [Click to show/hide]"
 
-    `The Movie Title (2010) - {edition-Ultimate Extended Edition} [IMAX HYBRID][Bluray-1080p Proper][3D][DV HDR10][DTS 5.1][x264]-EVOLVE`
+    `The Movie Title (2010) [imdbid-tt0066921]- {edition-Ultimate Extended Edition} [IMAX HYBRID][Bluray-1080p Proper][3D][DV HDR10][DTS 5.1][x264]-EVOLVE`
```

Comment:
Does this need an extra space after [imdbid-ttxxxxxxx] ?

Reply:
> Does this need an extra space after [imdbid-ttxxxxxxx] ?

you mean the space between the ID and the dash ?

Reply:
Damn i didn't even see that, sniper vision.

Reply:
Yep, so `[imdbid-tt0066921]- ` should be `[imdbid-tt0066921] - `.

Reply:
> Yep, so `[imdbid-tt0066921]- ` should be `[imdbid-tt0066921] - `.

darn you're right